### ISSUE TITLE
Max training step

### ISSUE THREAD

**Issue by ro6842** [tflearn/tflearn#588]
I am training some data, but the training step is stopped at 16777984, when I trained more epochs, the training steps don't change(but the total loss and accuracy are still running), so whether TFlearn has a max training step?
